api/DHSC.FingertipsNext.Api/Controllers/HealthMeasureController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using DHSC.FingertipsNext.Modules.Core.Service;
+using DHSC.FingertipsNext.Modules.Repository;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace DHSC.FingertipsNext.Api.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class HealthMeasureController : ControllerBase
+{
+
+    private readonly IHealthMeasureService healthMeasureService;
```

Comment:
Can we prefix private variables with an underscore please, e.g. _healthMeasureService